File: Java/Java/GCD.java
import java.util.*;

class GCD{
    public static void main(String []args){

        Scanner s = new Scanner(System.in);

        System.out.print("Enter First Number : ");
        int num1 = s.nextInt();
        System.out.print("Enter Second Number : ");
        int num2 = s.nextInt();
        
        int gcd = 0;

        for(int i=1;i<=num1 && i<=num2;i++){
            if(num1 % i == 0 && num2 % i == 0){
               gcd = i;
            }
        }

        System.out.println("GCD is : "+gcd);

        int lcm = (num1 * num2) / gcd;
        System.out.println("LCM is : "+lcm);

        while(num1 != num2){
            
            if(num1 > num2){
                
                num1 -= num2;
                //num1 = num1 - num2;
            }else{
                num2 -= num1;
                //num2 = num2 - num1;
            }
        }

        System.out.println("GCD is : "+num1);

        
    }
}

File: C/GCD.c
#include<stdio.h>

int main()
{
    int m,n,i,gcd;
    
    printf("Enter Two Number: ");
    scanf("%d %d",&m,&n);
    
    m = (m>0) ? m:-m;
    n = (n>0) ? n:-n;
    
    for(i=1;i<=m && i<=n;i++) 
    {
       if(m%i==0 && n%i==0)
       {
          gcd=i;
       }
    }
    
    printf("GCD of %d & %d is : %d",m,n,gcd);
    return 0;
}